wurst/_items/GemOfKnowledge.wurst

```diff
@@ -0,0 +1,18 @@
+package GemOfKnowledge
+
+import ItemObjEditing
+
+@compiletime function create_gemofknowledge()
+    new ItemDefinition('I03F', 'bspd')
+    ..setName("Gem of Knowledge")
+    ..setAbilities("A050,AIx1")
```

Comment:
Add a comment at the end of this line listing out the two names of these abilities.